tag.php

```diff
@@ -81,6 +65,23 @@
 			exit();
 		}
 
+		// phpcs:ignore WordPress.WP.GlobalVariablesOverride.Prohibited
```

Comment:
I've moved after the listing page implementation because we were getting `undefined` `offset` when the global posts var is overriden. It causes conflicts with the listing page core.
In fact, that wasn't necessary to run it at the top of the code.